usefulUtilities/bouncerZone/bouncerZone.js

```diff
@@ -55,140 +96,88 @@
-        rejectionLocation = userDataProperty.whitelist.rejectTeleportLocation;
-        userDataUsernameWhitelist = userDataProperty.whitelist && userDataProperty.whitelist.usernames || [];
+        rejectionLocation = userDataProperty.rejectTeleportLocation || "/0,0,0/0,0,0,0";
+        ticketMarketplaceID = userDataProperty.whitelist && userDataProperty.whitelist.marketplaceID || 
+            DEFAULT_USER_DATA.rejectTeleportLocation;
+        userDataUsernameWhitelist = userDataProperty.whitelist && userDataProperty.whitelist.usernames || 
+            DEFAULT_USER_DATA.whitelist.usernames;
+        allowAdmins = userDataProperty.whitelist && userDataProperty.whitelist.allowAdmins || 
+            DEFAULT_USER_DATA.whitelist.allowAdmins;
+        var bounceSoundURL = userDataProperty.bounceSound && userDataProperty.bounceSound.bounceSoundURL || 
+            DEFAULT_USER_DATA.bounceSound.bounceSoundURL;
+        bounceSoundVolume = userDataProperty.bounceSound && userDataProperty.bounceSound.bounceSoundVolume || 
+            DEFAULT_USER_DATA.bounceSound.bounceSoundVolume;
+
+        downloadedBounceSound = bounceSoundURL && SoundCache.getSound(bounceSoundURL) || null;
     }
 
     // Moves my avatar to `rejectionLocation`
     function bounceAvatarFromZone() {
         if (DEBUG) {
             print("Rejected from zone to: ", rejectionLocation);
         }
-        Window.location.handleLookupString(rejectionLocation);
-    }
-
-    // Returns true if my avatar is inside the bouncer zone, false otherwise.
-    var HALF = 0.5;
-    function avatarIsInsideBouncerZone() {
-        var properties = Entities.getEntityProperties(_entityID, ["position", "dimensions", "rotation"]);
-        var position = properties.position;
-        var dimensions = properties.dimensions;
-        
-        var avatarPosition = MyAvatar.position;
-        var worldOffset = Vec3.subtract(avatarPosition, position);
-
-        avatarPosition = Vec3.multiplyQbyV(Quat.inverse(properties.rotation), worldOffset);
-
-        var minX = 0 - dimensions.x * HALF;
-        var maxX = 0 + dimensions.x * HALF;
-        var minY = 0 - dimensions.y * HALF;
-        var maxY = 0 + dimensions.y * HALF;
-        var minZ = 0 - dimensions.z * HALF;
-        var maxZ = 0 + dimensions.z * HALF;
-
-        if (avatarPosition.x >= minX && avatarPosition.x <= maxX
-            && avatarPosition.y >= minY && avatarPosition.y <= maxY
-            && avatarPosition.z >= minZ && avatarPosition.z <= maxZ) {
-            
-            if (DEBUG) {
-                print("Avatar IS inside zone");
-            }
-            return true;
-
-        } else {
-            if (DEBUG) {
-                print("Avatar IS NOT in zone");
-            }
-            return false;
+        if (downloadedBounceSound) {
+            playSound(downloadedBounceSound, bounceSoundVolume, MyAvatar.position, true, false);
         }
+        Window.location.handleLookupString(rejectionLocation);
```

Comment:
```suggestion
        Window.location = rejectionLocation;
```

Reply:
We actually need to change this to use feet position, anyway, so...resolving and updating.

Reply:
As discussed in Slack, let's just keep this as `Window.location = rejectionLocation;` and move on.